### DISCUSSION TITLE
Trying to get all item inside Announcments lists using this Query Template "{?{searchTerms} -path:*/Announcements/*}" will return only one result

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2421] [Q&A] [answered]
I added the PnP modern search web part v4, and i define this Query Templete:-

`{?{searchTerms} -path=*/Announcements/*}`

the idea is that i want to get all the items inside the Announcments lists from multiple sites and sub-sites, but i am only getting one announcement item, any idea what is going on ?

**Comment by FranckyC** [comment, marked as answer]
Hi @mvcsharepointdev, wildcard character doesn't work as prefix, only suffix. You can try `{?{searchTerms} contentclass:STS_ListItem ParentLink:"Announcements/*" IsDocument:1}`

**Reply by mvcsharepointdev** [reply]
@FranckyC worked thanks a lot for your help
